KeyFrameDetector/key_frame_detector.py

```
@@ -4,94 +4,95 @@
 import time
 import peakutils
+import logging
 from KeyFrameDetector.utils import convert_frame_to_grayscale, prepare_dirs, plot_metrics
 
+logging.basicConfig(
+    filename='keyframe_detection.log',
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+
 def keyframeDetection(source, dest, Thres, max_keyframes=8, plotMetrics=False, verbose=False):
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)